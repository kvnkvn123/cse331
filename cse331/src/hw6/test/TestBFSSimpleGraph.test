# Test the BFS on a graph with multiple paths

LoadGraph simple simpleGraph3.tsv
ListNodes simple

FindPath simple a a
FindPath simple a b
FindPath simple b a
FindPath simple a c
FindPath simple c a
FindPath simple a e
FindPath simple g e
FindPath simple a h
FindPath simple g c