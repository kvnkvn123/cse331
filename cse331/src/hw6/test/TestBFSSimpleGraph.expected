# Test the BFS on a graph with multiple paths

loaded graph simple
simple contains: a b c e f g h i

path from a to a:
path from a to b:
a to b via 1
path from b to a:
b to a via 1
path from a to c:
a to c via 3
path from c to a:
c to a via 3
path from a to e:
a to b via 1
b to e via 6
path from g to e:
g to f via 8
f to b via 7
b to e via 6
path from a to h:
no path found
path from g to c:
g to f via 8
f to b via 7
b to a via 1
a to c via 3